packages/manager/modules/sms/src/sms/batches/pending/cancel/translations/Messages_fr_FR.json

```diff
@@ -0,0 +1,12 @@
+{
+  "sms_cancel_title": "Confirmation",
+  "sms_cancel_details_name": "Nom",
+  "sms_cancel_details_date": "Date de création",
+  "sms_cancel_details_sender": "Expéditeur",
+  "sms_cancel_details_receivers": "Destinataires",
+  "sms_cancel_details_message": "Message",
+  "sms_cancel_question": "Êtes-vous sûr de vouloir annuler la campagne affichée ci-dessous ?",
+  "sms_removing": "Annulation en cours…",
+  "sms_cancel_error": "Impossible d‘annuler la campagne. {{error}}",
+  "sms_cancel_success": "La campagne a été annulée"
```

Comment:
Don't find where theses translations are used.

Reply:
thanks for feedback, i've not found this one on any part of code. i've removed it on last commit :)